plugin-archetype/src/main/resources/archetype-resources/src/main/java/PluginEntry.java

```diff
@@ -0,0 +1,14 @@
+package ${package};
+
+import io.gomint.plugin.Plugin;
+import io.gomint.plugin.PluginName;
+import io.gomint.plugin.Startup;
+import io.gomint.plugin.StartupPriority;
+import io.gomint.plugin.Version;
+
+@PluginName("${artifactId}")
+@Version(major = 1, minor = 0)
+@Startup(StartupPriority.STARTUP)
+public class PluginEntry extends Plugin {
+
```

Comment:
Wouldn't it be nice to have all 'plugin lifecycle' methods already added? Init, enable and disable?

Reply:
Thats a general discussion to be had. Do we want to deliever more than MVP?

Reply:
```java
@Override
public void onInstall() {

}
```
A blank `onInstall` method. It doesn't seem like much but it's a teeny-tiny detail that enhances the _experience_. If it wasn't there I'd imagine a user going _”What?! You give me an entire prepared plugin in form of a Maven archetype and you leave out the most critical part of a plugin?”_ — This would be the same as if Cargo were to generate a new project but doesn't bother adding a `main.rs` file. I think a (blank) `onInstall` method is somewhat required.